modulefiles/modulefile.ProdGSI.jet

```diff
@@ -11,21 +11,23 @@
 #module-whatis " GDAS_ENKF whatis description"
 #
 #
-module load newdefaults
-module load intel/2016.2.181 
-module load impi/5.1.1.109
+
+module load intel/18.0.5.274
+module load impi
 #
-module load szip/2.1   
+module load szip/2.1
 module load hdf5/1.8.9
-module load netcdf4/4.2.1.1  
+module load netcdf4/4.2.1.1
```

Comment:
this netCDF shouldn't be used now, correct?

Reply:
Yes, removed.